### DISCUSSION TITLE
Accounts are getting suspended

### DISCUSSION THREAD

**Discussion by sudhagar-mr-17051** [mastodon/mastodon#29797] [General]
Hi team,
We have enabled bot option while creating a account, based on your documentation, it is stated that bot accounts will not be suspended and monitored, but our accounts are suspended, kindly assist us.

Our use case:
We integrated Mastodon API and we run automation to test if all the APIs working fine for every few hours, to make sure user does not face any inconvenience.

Kindly Assist us.
Also let us know for any API related issues, we would like to have a point of contact, every time while facing an issue, we are not receiving proper response in this discussion forum, kindly let us know if we want to report API related issues some where else.

Thanks in advance

**Comment by renchap** [comment]
Mastodon is an open-source software, and there are multiple thousand of instances. You need to contact the admins for the instance where you are creating your accounts, this place is for discussions about the open-source project.

On which instance are you creating your accounts?

**Comment by sudhagar-mr-17051** [comment]
Thanks for the explanation, incase if this forum is for discussions about the open-source project, where can I raise the queries regarding API for each instance. Kindly assist us with the response, so that we could proceed further as per your guidance.

**Reply by alizehctl** [reply]
@sudhagar-msr you should check the "About" page of your instance for that information. It is likely you will have to reach out to your instance admins via DM, email, or a public post if the information isn't already documented.

**Reply by sudhagar-mr-17051** [reply]
Thanks for the information
